lib/GeoExt/data/PrintProvider.js

```diff
@@ -403,87 +99,42 @@ GeoExt.data.PrintProvider = Ext.extend(Ext.util.Observable, {
             ]
         });
 
-        if(config.capabilities) {
-            this.loadStores();
-        } else {
-            if(this.url.split("/").pop()) {
-                this.url += "/";
-            }
-            this.initialConfig.autoLoad && this.loadCapabilities();
-        }
+        this.attributes = new Ext.data.JsonStore({
+            root: "attributes",
+            fields: ["name", "type"]
+        });
     },
 
-    /** api: method[setLayout]
-     *  :param layout: ``Ext.data.Record`` the record of the layout.
+    /** private: method[getCapabilitiesURL]
      *
-     *  Sets the layout for this printProvider.
+     *  Get capabilities URL.
      */
-    setLayout: function(layout) {
-        this.layout = layout;
-        this.fireEvent("layoutchange", this, layout);
+    getCapabilitiesURL: function() {
+        return this.url + "info.json";
```

Comment:
any specific reason why you got rid of the trailing slash check from the constructor?